.buildkite/env-scripts/util.sh

```diff
@@ -60,7 +60,7 @@ check_platform_architecture() {
   esac
 }
 
-retry() {
+retry_with_count() {
```

Comment:
`retry` is the default function name used in different places (repositories and even in this project), see https://github.com/elastic/beats/pull/38135/files#diff-71e4f7cc0d357183cbbb900bb97a29db006263fb1967d06035e417c38f27e53dR7

I'd say to keep it `retry`

Reply:
Changed that according to Dimitrios comment: https://github.com/elastic/beats/pull/37827#discussion_r1504301342

Reply:
Aha! Thanks.